alaoweb/2020.alaoweb.org
# Awards page

* partially addresses #112 (may still want to work up some "ads" for awards to appear on site after the ceremony)
* adds Awards page
* includes Awards page in header nav
* creates _awards.yml file -- currently populated with real award data and a couple fake award winners

## to test:
* check out [/awards/](https://deploy-preview-242--alao-production.netlify.app/awards/) (you'll see award descriptions with no winners
* change the showAwardWinners config to "true" to see an example of a page with two winners; one has a pic and one doesn't. (We can expect that we'll have some pics but not all, especially at first.)
